config/nginx/README.md

```diff
@@ -0,0 +1,28 @@
+
+## Features:
+* Docker image that contains static content and an nginx config file which does the app uri redirect for the api.
+* Stanza in docker-compose.yml defining the webserver service, and a link for the content directory to a new docker volume.
+* port 3000 of webserver container published to 3000 in docker-compose.dev.yml. I assume in prod the server would instead publish to 80 or 443, which are already exposed by the nginx  base image.
+* New make target called webserver_image that builds the docker image.
+* In the make file, added the webserver_image target to the dev_deploy target’s dependencies.
+* Missing feature: https configuration (and cert).
+* The Nginx docker image already redirects the server access and error logs to std_out and std_err, so docker logging captures both. 
+* The destination of static files in the image is just nginx's default directory for static files: /usr/share/nginx/html/ . 
+* Building the image requires that the frontend code has already been built in src/frontend/build, as described in src/frontend/README.md (just cd to src/frontend and run "npm run build"). I don't know the best workflow to put these steps together so it's just manual for now.
+
+## Behavior issue to be aware of:
+* After visiting the webpage, nginx will forward requests to the backend only if you do a hard refresh (shift-F5) on the endpoint url. Otherwise, that url first hits a react service which redirects to the site’s login page instead of reaching the server (because react doesn't recognize the endpoint uri). This stackoverflow post describes the issue:
+https://stackoverflow.com/questions/51120222/how-to-serve-react-app-and-django-blog-from-nginx
```

Comment:
Is this something that you are seeing in our app on the current `master` branch?

The SO post you linked says that the solution is probably to turn off Service Workers in the React app. I already did this https://github.com/CodeForPortland/recordexpungPDX/pull/104 so it's possible that merging that PR will fix the issue.

I don't totally understand the behavior you're describing; we should probably discuss this verbally at the next meeting.

Reply:
I find that with the app running on master (using npm start instead of nginx), every path on the website redirects to the login page rather than returning 404. I don’t know if that’s the intended behavior. But without running it on the nginx server which is supposed to redirect paths matching api/*, the comparison doesn't mean much.

I agree, let’s check in about this in person.

Reply:
I don't think that's intended behavior but it's probably expected behavior given the way the app is set up now.